scripts/yum-wrapper

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+function hr() {
+  printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+}
+
+function banner() {
+  hr
+  echo -e "Installing additional packages on the root filesystem might exceed the allocated overlay size and lock the Minishift VM. Proceed with the installation at your own risk."
```

Comment:
I am not sure about the word `overlay size`. Not sure what is supposed to mean. @thatdocslady @gbraad can you please explain it a little.